assets/js/dark-mode.js

```diff
@@ -2,14 +2,14 @@ const toggleContainer = document.querySelector('#toggle-container');
 const inputCheck = document.querySelector('input[type="checkbox"]');
 const content = document.querySelector('.hero_content');
 const contentPara = document.querySelector('.hero_content > p');
-const contenttitle = document.querySelector('.hero_content > h2');
+const contenttitle = document.querySelector('#hero-heading');
```

Comment:
updated this because of the addition of an id "hero-heading" to the h2 element in the hero section of the website in index.html